flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/nodes/logical/FlinkLogicalTableSourceScan.scala

```diff
@@ -81,7 +81,13 @@ class FlinkLogicalTableSourceScan(
     val terms = super.explainTerms(pw)
         .item("fields", tableSource.getTableSchema.getFieldNames.mkString(", "))
 
-    val sourceDesc = tableSource.explainSource()
+    val auxiliarySourceDesc = tableSource match {
+      case fts: FilterableTableSource[_] =>
+        s"FilterPushDown=${fts.isFilterPushedDown.toString}"
```

Comment:
`s"FilterPushDown=${fts.isFilterPushedDown.toString}"` is not the digest of a `FilterableTableSource `,  consider the following case: there are several different filters to be pushed down. 
so, i think the digest should be the string value of the pushed filters.

Reply:
hmm. I think this is valid. I original think since there could only be one "match" and "onMatch", all the different filters either can be pushed down all cannot (e.g. always the same set of filter being push down).

but I think yes, there's nothing preventing calcite to choose to partially push down (not partially possible, but partially push down some of the filter because of volcano planner thinks it is more efficient than another). I doubt this is possible at this moment.